.eslintignore

```diff
@@ -1,6 +1,6 @@
 *.template.js
-/src/Powercord/plugins/*
-!/src/Powercord/plugins/pc-*
+/plugins/*
+!/plugins/pc-*
```

Comment:
Should add themes here too, idk why it wasn't there before

Reply:
pc- is no longer needed anymore.
Also, don't think these are supposed to have a leading slash